<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1144</width>
    <height>759</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ETUPaint</string>
  </property>
  <property name="windowIcon">
   <iconset resource="QResource.qrc">
    <normaloff>:/resources/icon.png</normaloff>:/resources/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0" colspan="2">
     <widget class="graphicsView" name="view"/>
    </item>
    <item row="0" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QToolButton" name="lineButton">
        <property name="toolTip">
         <string>Draw a line</string>
        </property>
        <property name="statusTip">
         <string>Draw a line</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="QResource.qrc">
          <normaloff>:/resources/paintbrush.png</normaloff>:/resources/paintbrush.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="lineThick">
        <property name="toolTip">
         <string>Line thickness</string>
        </property>
        <property name="minimum">
         <number>2</number>
        </property>
        <property name="maximum">
         <number>20</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="triangleButton">
        <property name="toolTip">
         <string>Draw a triangle</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="QResource.qrc">
          <normaloff>:/resources/triangleButton.png</normaloff>:/resources/triangleButton.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="triangleThick">
        <property name="toolTip">
         <string>Lines thikness</string>
        </property>
        <property name="minimum">
         <number>2</number>
        </property>
        <property name="maximum">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="collageButton">
        <property name="toolTip">
         <string>Build a collage</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="QResource.qrc">
          <normaloff>:/resources/collage.png</normaloff>:/resources/collage.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="mSizeCollage">
        <property name="toolTip">
         <string>Y amount</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>20</number>
        </property>
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="nSizeCollage">
        <property name="toolTip">
         <string>X amount</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>20</number>
        </property>
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="repaintButton">
        <property name="toolTip">
         <string>Repaint the biggest rectangle</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="QResource.qrc">
          <normaloff>:/resources/repaintRect.png</normaloff>:/resources/repaintRect.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="labelCoords">
      <property name="text">
       <string>x y</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1144</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave_as"/>
    <addaction name="actionInfo"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAuthor"/>
    <addaction name="actionFAQ"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open..</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save..</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionInfo">
   <property name="text">
    <string>Image Properites</string>
   </property>
  </action>
  <action name="actionAuthor">
   <property name="text">
    <string>Author</string>
   </property>
  </action>
  <action name="actionFAQ">
   <property name="text">
    <string>FAQ..</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>graphicsView</class>
   <extends>QGraphicsView</extends>
   <header>inc/graphicsview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="QResource.qrc"/>
 </resources>
 <connections/>
</ui>
